reaktive/src/commonMain/kotlin/com/badoo/reaktive/disposable/Various.kt

```diff
@@ -1,16 +1,14 @@
 package com.badoo.reaktive.disposable
 
-import com.badoo.reaktive.utils.atomic.AtomicBoolean
+import kotlin.js.JsName
 
-inline fun disposable(crossinline onDispose: () -> Unit = {}): Disposable =
-    object : Disposable {
-        @Suppress("ObjectPropertyName") // Backing property
-        private var _isDisposed = AtomicBoolean()
-        override val isDisposed: Boolean get() = _isDisposed.value
+@Deprecated(
+    message = "Use Disposable instead",
+    replaceWith = ReplaceWith("Disposable(onDispose)", "com.badoo.reaktive.disposable.Disposable")
+)
+@JsName("disposableDeprecated")
```

Comment:
It is breaking change. You will break current JS code.

Reply:
I'm still not 100% sure what exactly it breaks. I have checked our sample-js-app - the deprecated method is still visible and available as `disposable` and marked as deprecated.

Reply:
If someone uses `disposable` in js code, it won't work anymore. Even if no one uses it, in future it is possible, should be careful with such changes.

Reply:
Should I revert the name change? AFAIK Kotlin functions with arguments are exported with mangled names like 'repeat_j15sdp'.

Reply:
Keep it, you can mention name change in changelog.